src/types.zig

```diff
@@ -1815,7 +1818,7 @@ test "getJNIVersion" {
     var env = getTestingJNIEnv();
 
     const version = env.getJNIVersion();
-    try testing.expectEqual(JNIVersion{ .major = 10, .minor = 0 }, version);
+    try testing.expectEqual(JNIVersion{ .major = 19, .minor = 0 }, version);
```

Comment:
This is the version type I changed. It seems odd to tie the test to a specific JNIVersion

Reply:
I removed this change and that fixed the build failure.